images/gitlab-sidekiq/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-sidekiq-ee:v14.4.1
+FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-sidekiq-ee:v14.6.3
 
 RUN ss='s/return false unless Feature.enabled?(:ci_yaml_limit_size.*' \
   ; ss="$ss/return false/g" \
```

Comment:
I'm curious if these are still working, do you have any idea if these patch still do anything?

Reply:
I tested these builds locally to make sure that the patch at least applies.  Will have to test in staging to see if it works.